osf/metrics.py

```diff
@@ -124,6 +124,10 @@ class BasePreprintMetric(MetricMixin, metrics.Metric):
     preprint_id = metrics.Keyword(index=True, doc_values=True, required=True)
     version = metrics.Keyword(index=True, doc_values=True)
     path = metrics.Text(index=True)
+    referrer = metrics.Text(index=True)
+    user_agent = metrics.Text(index=True)
+    origin = metrics.Text(index=True)
+    uri = metrics.Text(index=True)
```

Comment:
@erinspace / @sloria 
I added new fields here and ran `python manage.py sync_metrics` and it doesn't seem to have any effect, you can still query properly in the shell, etc. Just want to get out of the way that I'm not exactly sure what we're using this for, but it's in the ticket so I assume it's necessary for ES.

Reply:
@Johnetordoff `sync_metrics` should update the [index templates](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html). You can view them using `httpie`:

```
http :9201/_template/
```